lang/lang-import.sh

```diff
@@ -351,8 +352,74 @@ fi
 #fi
 
 #replace in slovak translation
-#if [ "$LNG" = "sk" ]; then
-#fi
+if [ "$LNG" = "sk" ]; then
```

Comment:
I'm curious, how many of these characters are used in the translation overall? I'd assume not all capital letters are used at least

Reply:
Resolve this as it is a question.